### ISSUE TITLE
[REC] Shaman Enhancement - Feral Spirit locks the queue on weapon imbues

### ISSUE THREAD

**Issue by ivannpaz** [Hekili/hekili#1793]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Describe the Issue

When summoning the `Feral Spirit`, queue changes and locks on suggesting the weapon imbues:

![image](https://user-images.githubusercontent.com/16879/198837160-08d71228-3dc2-4373-9d4f-c26524f97f88.png)

In the above case, those are not even the bindings I have for weapon imbuing.

### How to Reproduce

My current context:

- Talents: BcQAfcj78nJtvjmejSqe5Zhm9IRSaRCiIhgAtEJAAAAAAAAAAAAA0iQES0CRhgEJlAUSAA
- Legendaries: Unity on neck, Fiery Skybreaker's on ring2
- Weapons already have both `Windfury Weapon` and `Flametongue Weapon` applied.

**Steps to reproduce:**

1. Summon Feral Spirit at any point in the rotation (even at the beginning and not actually engaging on combat)

It only resets if doing a `/reload` in game.

### Snapshot (Link)

https://pastebin.com/h7WKWJ93

**Comment by Hekili** [comment]
> In the above case, those are not even the bindings I have for weapon imbuing.

Yeah, keybinds are a backburner issue because the default UI actionbars and paging got changed very late in beta/PTR.

The snapshot you provided shows the following recommendations:
`Primordial Wave(0.00), Stormstrike(1.19), Lava Lash(2.38), Sundering(3.57), Fire Nova(4.76)`

Did you take the snapshot while you were experiencing the issue you described?  It doesn't match the screenshot you provided.

**Comment by Hekili** [comment]
I triggered the error myself and found the issue.  Will post another release in an hour or two.  If you want to fix for yourself in the meantime, on line 862 of Dragonflight/ShamanEnhancement.lua, change `chain_lighting` to `chain_lightning`.  Typo.

**Comment by ivannpaz** [comment]
> Did you take the snapshot while you were experiencing the issue you described? It doesn't match the screenshot you provided.

I just pressed my keybind for `Feral Spirits` and then pressed "Pause" in the Hekili config window. Chances are I did the snapshotting incorrectly, so sorry about that!

Just tried again with the latest update from CurseForge and the error still persists. Took another snapshot that looks more "correct" on what the current state was for the decision queue here: https://pastebin.com/M0FGjC2u

Then went also into the aforementioned file, and corrected the typo (thought that the latest release you had would contain it)., updated the line as suggested, and now it all works fine.

An unrelated issue, is that I do not get suggestions for `Feral Spirit` or `Chain Harvest`, or that the queue starts with `Primordial Wave` up front, but I guess this has more to do with the simc apls?

Needless to say, thank you for your time! Awesome work!
